db/migrate/20191113122435_rename_order_to_ordinal_on_reference.rb

```diff
@@ -0,0 +1,5 @@
+class RenameOrderToOrdinalOnReference < ActiveRecord::Migration[6.0]
```

Comment:
Can we remove the migrations to add/remove `ordinal?` - they haven't been run on prod so would be a no-op.